api_proxy.yaml

```diff
@@ -488,92 +486,40 @@ definitions:
     required:
       - platform
       - pushChannel
-  ProfileWithEmail:
+
+  Profile:
     type: object
-    title: Profile with wmail
-    description: Describes the user's profile.
+    title: Profile
+    description: Profile
     properties:
-      blocked_inbox_or_channels:
-        $ref: "#/definitions/BlockedInboxOrChannels"
-      email:
-        $ref: '#/definitions/EmailAddress'
-      family_name:
-        type: string
-      fiscal_code:
-        $ref: '#/definitions/FiscalCode'
-      has_profile:
-        $ref: "#/definitions/HasProfile"
-      is_email_set:
-        $ref: "#/definitions/IsEmailSet"
-      is_inbox_enabled:
-        $ref: "#/definitions/IsInboxEnabled"
-      is_webhook_enabled:
-        $ref: "#/definitions/IsWebhookEnabled"
-      name:
-        type: string
-      spid_email:
-        $ref: '#/definitions/EmailAddress'
-      spid_mobile_phone:
-        type: string
-        minLength: 1
-      preferred_languages:
-        type: array
-        items:
-          $ref: '#/definitions/PreferredLanguage'
-      version:
-        $ref: '#/definitions/Version'
+      spid_data:
+        $ref: "#/definitions/ProfileSpidData"
+      cd_data:
```

Comment:
not easy to find a name here - `cd` is the italian acronym, perhaps we can use a name consistent the type: `extended`